### ISSUE TITLE
[Usability] User experiences an unexpected interaction with a component.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#78948]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Identity Core Team
**Product:** Terms of Use (ToU)
**Feature:** Terms of Use (ToU)

## Findings details
**VA.gov Experience Standard - issue:** User experiences an unexpected interaction with a component.
**VA.gov Experience Standard - category:** Usability
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When you're authenticated, the Accept and Decline buttons at the bottom of the page each have an `aria-label` to clarify the meaning of the buttons for screen reader users:
- Accept, `aria-label` of "I accept the VA online services terms of use"
- Decline, `aria-label` of "I decline the VA online services terms of use"

However, this improvement for screen reader users introduces a potential barrier for voice command software users.

When a voice command user interacts with a button or link on the page, the user typically says "Click [name of element]." Sighted voice command users will not see the `aria-label` since it doesn't appear on screen, so they will likely say the name of the element that's visible, eg. "Click Accept."

After hearing that command, their software then searches the page for a match --- meaning, searches the page for something with the name "Accept." But voice command software searches for matches in the browser's accessibility tree, and the `aria-label` value replaces the visible text in the accessibility tree.

That means there won't be any exact matches for "Accept" or "Decline" since those buttons aren't named "Accept" or "Decline" anymore.

Just about all voice command software will then try to find a likely partial match. A higher-end product like Dragon NaturallySpeaking will probably be smart enough to identify "I accept the VA online services terms of use" as a partial match and click the button. Lower-quality products will sometimes only register a partial match if there's a match at the start of the name in the accessibility tree. In other words, some voice command software will hear "accept" and not identify "I accept" as a match.

Most voice command users will have a workaround that should work here, but it's a point of friction that would be good to try to avoid.

## Recommended action
A couple of possible fixes:
- I'm not totally certain the `aria-label` is necessary on these buttons. "Accept" and "Decline" as verbs don't make a ton of sense in isolation so it would usually make sense to try to provide context about what you're accepting/declining in the `aria-label` for screen reader users. But in the context of this page I think there's a good case that screen reader users will know what the buttons mean without the extra words.
- Alternatively, changing the `aria-label` text to just "Accept the VA online services terms of use" (no "I") would be fine here too. It should be a better match for voice command software and the "I" doesn't seem grammatically necessary for understanding the intent of the buttons.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
